### DISCUSSION TITLE
close immortal pr

### DISCUSSION THREAD

**Discussion by icecream17** [renovatebot/renovate#20483] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

close https://github.com/icecream17/solver/pull/214

### What you tried so far.

the link says "Get config help if undesired"

### Relevant debug logs

no logs

**Comment by rarkins** [comment, marked as answer]
If `@testing-library/react` were removed from that monorepo then it would all be v18 and would be non-immortal. You have grouped it due to this package rule: https://github.com/icecream17/solver/blob/70ad675a498f3890e951fa1214b0956424399567/renovate.json#L6-L9

I suggest you remove that package rule so that react components are grouped by their monorepo only
